packages/algo-ts/src/op.ts

```diff
@@ -208,7 +208,7 @@ export const AppGlobal = {
 
   /**
    * global state of the key A in the current application
-   * @param state key.
+   * @param a state key.
    *  * @return value. The value is zero (of type uint64) if the key does not exist.
```

Comment:
```suggestion
   * @return value. The value is zero (of type uint64) if the key does not exist.
```

Can we fix the extra `*` here?